content-test/addon/PrefsProvider.test.js

```diff
@@ -4,31 +4,32 @@ const {SimplePrefs} = require("shims/sdk/simple-prefs");
 describe("PrefsProvider", () => {
   let prefsProvider;
   let simplePrefs;
+  let tabTracker = {};
 
   // This is just a utility to mock required options for PrefsProvider.
   // Override as necessary with the setup function.
   function setup(customPrefs = {}) {
+    tabTracker.handleUserEvent = sinon.spy();
     simplePrefs = new SimplePrefs(customPrefs);
     const {PrefsProvider} = createPrefsProvider({"sdk/simple-prefs": simplePrefs});
-    prefsProvider = new PrefsProvider({
+    prefsProvider = new PrefsProvider(tabTracker, {
       broadcast: sinon.spy(),
       send: sinon.spy()
     });
   }
 
   beforeEach(() => setup());
 
-  it("should set instance properties", () => {
-    assert.property(prefsProvider, "broadcast");
-    assert.property(prefsProvider, "send");
-  });
+  it("should set instance properties", () => (
+    assert.property(prefsProvider, "broadcast")
```

Comment:
nit: why change this from `{ statement; }` to `(expression)`? Seems like it'll be easy to run into errors/confusion.